server/cmd/mmctl/commands/config.go

```diff
@@ -128,6 +137,9 @@ func init() {
 	ConfigSubpathCmd.Flags().StringP("path", "p", "", "path to update the assets with")
 	_ = ConfigSubpathCmd.MarkFlagRequired("path")
 
+	ConfigExportCmd.Flags().Bool("remove-masked", false, "remove masked values from the exported configuration")
```

Comment:
Should this default to true to be safe by default?

Reply:
This is a separate command right? Or are we changing existing behavior somewhere?

Reply:
Yep this is a new command, but using an existing API. We can set to `true` to avoid errors.

Reply:
This doesn't seem to have been updated to default `true`